src/components/PluginSettings/index.tsx

```diff
@@ -243,6 +245,26 @@ export default ErrorBoundary.wrap(function Settings() {
         );
     };
 
+    const [newPlugins, , newPluginsLoading] = useAwaiter(() => DataStore.get("Vencord_existingPlugins").then((cachedPlugins: Record<string, number>) => {
```

Comment:
this get will return undefined if there is no entry which will later on read to a read of undefined.foo which will lead to a promise rejection. Please handle this case

Reply:
why did you mark this as resolved? this is still an issue

Reply:
```suggestion
    const [newPlugins, , newPluginsLoading] = useAwaiter(() => DataStore.get("Vencord_existingPlugins").then((cachedPlugins: Record<string, number> | undefined) => {
```

if it is true that it can be undefined, then it should be typed accordingly